### PR TITLE
Improve Servicebus log

### PR DESCRIPTION
**Pull request by warrenzhu25**

### CODE REVIEW

File: spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/ServiceBusTemplate.java [hunk 1]
```diff
@@ -68,6 +62,13 @@ public ServiceBusTemplate(@NonNull T senderFactory) {
         return this.senderFactory.getOrCreateSender(destination).sendAsync(serviceBusMessage);
     }
 
+    public void setCheckpointConfig(CheckpointConfig checkpointConfig) {
+        Assert.state(isValidCheckpointConfig(),
+                "Only MANUAL or RECORD checkpoint mode supported in ServiceBusTemplate");
```
Comment (xscript) [inline spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/ServiceBusTemplate.java]: `supported` -> `is supported`

File: spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/queue/ServiceBusQueueTemplate.java [hunk 2]
```diff
@@ -92,5 +94,18 @@ public QueueMessageHandler(Consumer<Message<U>> consumer, Class<U> payloadType,
         protected CompletableFuture<Void> failure(UUID uuid) {
             return queueClient.abandonAsync(uuid);
         }
+
+        @Override
+        protected String buildCheckpointFailMessage(Message<?> message) {
+            String failMsg = "Failed to checkpoint %s in queue '%s'";
```
Comment (xscript) [inline spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/queue/ServiceBusQueueTemplate.java]: Make it `static final`?

File: spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/queue/ServiceBusQueueTemplate.java [hunk 3]
```diff
@@ -92,5 +94,18 @@ public QueueMessageHandler(Consumer<Message<U>> consumer, Class<U> payloadType,
...
+            return String.format(failMsg, message, queueClient.getQueueName());
+        }
+
+        @Override
+        protected String buildCheckpointSuccessMessage(Message<?> message) {
+            String successMsg = "Checkpointed %s in queue '%s' in %s mode";
```
Comment (xscript) [inline spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/queue/ServiceBusQueueTemplate.java]: `static final`?

File: spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/converter/ServiceBusMessageConverter.java [hunk 4]
```diff
@@ -66,10 +68,10 @@ protected void setCustomHeaders(MessageHeaders headers, IMessage serviceBusMessa
     }
 
     @Override
-    protected MessageHeaders buildCustomHeaders(IMessage serviceBusMessage) {
+    protected Map<String, Object> buildCustomHeaders(IMessage serviceBusMessage) {
         Map<String, Object> headers = new HashMap<>();
 
-        headers.put(MessageHeaders.ID, UUID.fromString(serviceBusMessage.getMessageId()));
+        headers.put(AzureHeaders.RAW_ID, UUID.fromString(serviceBusMessage.getMessageId()));
```
Comment (xscript) [inline spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/converter/ServiceBusMessageConverter.java]: Do we need a test case to cover it?

File: spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/ServiceBusTemplate.java (hunk 1 above)
Comment (warrenzhu25) [reply spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/ServiceBusTemplate.java]: Yes

File: spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/converter/ServiceBusMessageConverter.java (hunk 4 above)
Comment (warrenzhu25) [reply spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/converter/ServiceBusMessageConverter.java]: Will have a test to cover this.

File: spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/queue/ServiceBusQueueTemplate.java (hunk 2 above)
Comment (warrenzhu25) [reply spring-integration-azure/spring-integration-servicebus/src/main/java/com/microsoft/azure/spring/integration/servicebus/queue/ServiceBusQueueTemplate.java]: Yes

**Pull request merged**